### PR TITLE
[13.0][MIG] stock_barcodes_gs1_expiry

### PR DESCRIPTION
**Pull request by jimhoefnagels**

### CODE REVIEW

**Comment by sergio-teruel** [comment]
@jimhoefnagels Can you rebase from 13.0 to solve conflicts??

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+stock-logistics-barcode https://github.com/Tecnativa/stock-logistics-barcode 13.0-mig-stock_barcodes
```
Comment (sergio-teruel) [inline oca_dependencies.txt]: this module already merged

File: oca_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+stock-logistics-barcode-2 https://github.com/Tecnativa/stock-logistics-barcode 13.0-mig-stock_barcodes_gs1
```
Comment (sergio-teruel) [inline oca_dependencies.txt]: This is https://github.com/OCA/stock-logistics-barcode/pull/254

**Review by etobella — APPROVED** [review]
No major changes

**Review by albariera — APPROVED** [review]
Technical review LGTM :+1:

**Comment by etobella** [comment]
@sergio-teruel All changes have been applied

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**